lib/TelegramHandlers/TgDocumentHandler.js

```diff
@@ -27,28 +28,41 @@ class TgDocumentHandler {
      * 
      * @param {*} from - Object that contains the information about the user name.
      * @param {*} document - The information about the document.
+     * @param {string} caption - The caption of the document, if any
      */
-    async RelayDocumentMessage(from, document) {
-        if (!this.Enabled) {
-            return;
-        }
-
+    async RelayDocumentMessage(from, document, caption) {
         let username = TgHelpers.ResolveUserName(from);
-
         let url = await this._tgbot.getFileLink(document.file_id);
 
-        let message = username +
-                        " Posted File: " +
-                        document.file_name + 
-                        " (" + 
-                        document.mime_type +
-                        ", " + 
-                        document.file_size +
-                        " bytes): " +
-                        url;
-
+        let message = this._GetDocumentMessage(caption, username, url);
         this._action(message);
     }
+
+    /**
+     * Generates the message to send out
+     * @param {string} caption - The caption of the document
+     * @param {string} from - User who sent the photo
+     * @param {string} fileUrl - URL to the document
+    */
+    _GetDocumentMessage(caption, from, fileUrl) {
+        var message;
+
+        if (Helpers.IsNullOrUndefined(caption)) {
+            caption = "Untitled Document";
+        }
+
+        if (!this.Enabled) {
```

Comment:
I understand what you're doing here, but I want to confirm this is the behaviour we want since it's different from other handlers. From what I'm aware, other handlers don't send anything if they're disabled, whereas this one will send a notice that a document was sent without forwarding the document.

If we're cool with that, then so am I, it just seems inconsistent at first look and we should make sure it's documented.

Reply:
@thenaterhood yes, this is confirmed behavior, and is documented. 

Thanks for pointing that out!